install/deb/templates/mail/nginx/default.stpl

```diff
@@ -1,5 +1,6 @@
 server {
-    listen      %ip%:%proxy_ssl_port% ssl http2;
+    listen      %ip%:%proxy_ssl_port% ssl;
+    http2 on;
```

Comment:
http2 on; breaks on any setup with Nginx < 1.25.1 

As we can't guarantee  Nginx 1.25.1 is installed